package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tern-gas",
   "version": "0.0.1",
   "description": "A plugin of tern for Google Apps Script.",
-  "main": "gas.js",
+  "main": "gas-2018.js",
```

Comment:
Is this necessary? Do staying`gas.js` have any problem?

Reply:
No problem, can stay as `gas.js`